scripts/container_tools/foxglove_bridge

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+ros2 launch foxglove_bridge foxglove_bridge_launch.xml \
+    send_buffer_limit:=1000000000 num_threads:=4
```

Comment:
I left the argument to increase parallel execution. My reference is this quote from the Foxglove bridge doc

`
num_threads – Number of threads to use for the ROS node executor. This controls the number of subscriptions that can be processed in parallel. 0 means one thread per CPU core. Defaults to 0.
`

Up to some extend, the bigger the better?

Reply:
I have not tested it, I just left it to 0, but we should indeed play around with it since it could have been the reason we experienced issues with Foxglove viz.
Let's put it to 4 and see if we can play with the value in future.